egs/iam/s5/local/unk_arc_post_to_transcription.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+import numpy as np
+from scipy import misc
+parser = argparse.ArgumentParser(description="""uses phones to convert unk to word""")
```

Comment:
It is good to add extended description for this function and its arguments. This script can be used in other applications.